tests/constraints.txt

```diff
@@ -6,6 +6,7 @@ Jinja2>=2.11.2
 pandas>=1.3.5
 pymongo>=3.12.0
 PyYAML>=5.4.1
+onnxruntime>=1.0.0,<=1.15.1
```

Comment:
Seems like if the version **is not pinned**, tests on Win are failing, but if the version **is pinned**, tests on Linux are failing.